packages/react/src/components/AccountSettings/ChangePassword/defaultComponents.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Alert, Button, PasswordField } from '../../../primitives';
+import {
+  AccountSettingsError,
+  AccountSettingsPasswordField,
+  AccountSettingsSubmitButton,
+} from '../types';
+
+/** ChangePassword subcomponents */
+// TODO: enable component override
+export const DefaultCurrentPassword: AccountSettingsPasswordField = (props) => {
+  return <PasswordField {...props} />;
+};
+
+export const DefaultNewPassword: AccountSettingsPasswordField = (props) => {
+  return <PasswordField {...props} />;
+};
+
+export const DefaultSubmitButton: AccountSettingsSubmitButton = (props) => {
+  const { children, ...rest } = props;
+  return <Button {...rest}>{children}</Button>;
+};
+
+export const DefaultError: AccountSettingsError = (props) => {
+  const { errorMessage, ...rest } = props;
```

Comment:
Since errorMessage is just the `children` we pass to the underlying `Alert`, we can just remove the `errorMessage` prop and just `children` as a passthrough prop:
```
// usage example in ChangePassword
<DefaultError>{error}</DefaultError>
```

Reply:
I specifically had `errorMessage` as a prop so that customer may do any errorMessage handling (like rewording). That said, I do see this is an antipattern to other default component here. (and same argument can be made with `DefaultSubmitButton). 

What do you think?

Reply:
Couldn't they just do something like:
```
// not sure what the component slot name is so using `Error`
<ChangePassword components={{ Error: ({ children, ...props }) => <ChangePassword.Error {...props}>{reformatError(children)}</ChangePassword.Error> />
```

Reply:
You're right, adjusting.

Reply:
Done in 4ea9fa43c00ef82f77c574bf9d7c45a3ed55b435